apps/web/src/stores/editor-store.ts

```diff
@@ -467,9 +467,13 @@ class EditorStore extends BaseStore<EditorStore> {
             event.item.fromType === "tag" &&
             event.item.toType === "note"
           ) {
-            updateSession(event.item.toId, undefined, {
-              tags: await db.notes.tags(event.item.toId)
-            });
+            for (const session of sessions) {
+              if ("note" in session && session.note.id === event.item.toId) {
+                updateSession(session.id, undefined, {
+                  tags: await db.notes.tags(event.item.toId)
+                });
+              }
+            }
```

Comment:
There's no need to loop over the sessions here. We are already looping over them above.
```suggestion
            updateSession(session.id, undefined, {
              tags: await db.notes.tags(event.item.toId)
            });
```

Reply:
`session` is undefined here. We need to loop over the sessions

Reply:
You're right. Can you also change it for colors a little bit above? I think it's also broken.

Reply:
@thecodrr done